uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1033.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+trait CRM_HRCore_Upgrader_Steps_1033 {
+
+  /**
+   * Updates menu permissions
+   *
+   * @return bool
+   */
+  public function upgrade_1033() {
+    $permissions = [
+      'access root menu items and configurations',
+      'edit system workflow message templates',
+      'edit user-driven message templates'
+    ];
+
+    $this->up1033_updateMessageTemplatePermissions($permissions);
+    $this->up1033_updateCommunicationsPermissions($permissions);
+
+    return TRUE;
+  }
+
+  /**
+   * Updates message template permissions
+   *
+   * @param array $permissions
+   */
+  private function up1033_updateMessageTemplatePermissions($permissions) {
+    civicrm_api3('Navigation', 'get', [
+      'parent_id' => 'Communications',
+      'url' => 'civicrm/admin/messageTemplates?reset=1',
```

Comment:
Do you think it would be safer to search by navigation name instead of something that might change like url?

I noticed that in `up1033_updateCommunicationsPermissions` you use `name`

Reply:
Sure. It will be safer. I will update 👌